plugins/woocommerce-blocks/assets/js/blocks/product-filters/block.json

```diff
@@ -9,8 +9,22 @@
 	"keywords": [ "WooCommerce" ],
 	"supports": {
 		"align": true,
+		"__experimentalBorder": {
+			"color": true,
+			"radius": true,
+			"style": true,
+			"width": true
+		},
```

Comment:
We usually tried to avoid using `__experimental` APIs for global styles, so if at some point Gutenberg changed or retired them, the styles of existing stores would not be impacted. :thinking:

Reply:
I noticed this "__experimentalBorder" is being used in other blocks as well, is there any alternative to add the border settings to a block? I tried changing to just "border" but it does not seem to have any effect in the Editor.

Reply:
Good point... I think we should avoid adding `__experimental` styles as much as possible, but given that `__experimentalBorder` is being used in many other blocks, let's go with it.